device_doctor/lib/src/android_device.dart

```diff
@@ -0,0 +1,231 @@
+// Copyright 2020 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
```

Comment:
Let's add documentation about the expected dependencies. Eg. this class depends on adb that needs to be shipped independently of this binary.